_includes/guide/endpoint_alert.html

```diff
@@ -1,12 +1,11 @@
 <p>
-  The /Patient and /Group endpoints have a key difference in how new beneficiaries are treated. New beneficiaries are those that are newly assigned to you since your last attribution date.
+	Using the _since parameter with the /Patient endpoint will return claims with a service date after the date provided. This includes claims for existing and newly attributed enrollees, those who have been assigned to your model entity since your last attribution date.
```

Comment:
claims loaded into CMS system? 
That'll be distinct from the actual service date.

Reply:
can you elaborate on how you would change the wording to be more explicit?